201-web-app-asev2-create/azuredeploy.parameters.json

```diff
@@ -1,21 +1,18 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json",
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json",
     "contentVersion": "1.0.0.0",
     "parameters": {
         "aseName": {
-            "value": "GEN_UNIQUE"
+            "value": "GEN-UNIQUE"
         },
         "aseLocation": {
-            "value": "South Central US"
+            "value": "changeme"
         },
-        "existingVirtualNetworkName": {
-            "value": "GET-PREREQ-virtualNetwork"
+        "existingSubnetResourceId": {

```

Comment:
use the prereq values in the master branch (i.e. revert this change)

Reply:
Can't do that.  This is the resource ID of the subnet.  It is not the VNet.  It is also a resource ID and not a name. 
If you were to make a resource ID from the various GEN values you could have: /subscriptions/SUBSCRIPTION ID/resourceGroups/GEN-VNET-RESOURCEGROUP-NAME/providers/Microsoft.Network/virtualNetworks/GEN-VNET-NAME/subnets/GEN-VNET-SUBNET1-NAME

I doubt that would work with your validators but do you want that instead?  

or do you have something that would be a reference to the resource ID of the subnet?

Reply:
I will try GEN-VNET-SUBNET-RESOURCE-ID and see if you are cool with that  :-)

Reply:
if it passes CI/CD - should be fine